app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title
+      base_title = t(:title)
```

Comment:
` t(:title)` to `I18n.t "title"`

note:
helper file and model file: you must use `I18n.t` (full syntax of I18n)
controller file and view layer (html): you can `t ....` (sort syntax of I18n)


you can read more this link https://github.com/framgia/coding-standards/blob/master/eng/rails/standard.md

Reply:
noted and done.